Formula/gnirehtet.rb

```diff
@@ -29,6 +30,12 @@ class Gnirehtet < Formula
   def install
     resource("java_bundle").stage { libexec.install "gnirehtet.apk" }
 
+    # Building the binary with rust 1.64.0 or later results in an error when running `gnirehtet relay`.
+    # ERROR Main: Execution error: IO error: Address family not supported by protocol family (os error 47)
+    # Issue ref: https://github.com/Genymobile/gnirehtet/issues/475
+    system "rustup-init", "-qy", "--no-modify-path", "--default-toolchain", "1.63.0"
+    ENV.prepend_path "PATH", HOMEBREW_CACHE/"cargo_cache/bin"
```

Comment:
If anyone knows of an upstreamable fix for Rust 1.64.0+, then we can patch it in instead.

For now, went with workaround of downgrading Rust version.